echopype/tests/echodata/test_echodata.py

```diff
@@ -317,17 +317,17 @@ def test_getattr(self, converted_zarr):
             'provenance': 'Provenance',
             'sonar': 'Sonar',
             'beam': 'Sonar/Beam_group1',
-            'vendor': 'Vendor',
+            'vendor': 'Vendor_specific',
         }
         for group, path in expected_groups.items():
             ds = getattr(ed, group)
             assert ds.equals(ed[path])
 
     def test_setitem(self, converted_zarr):
         ed = EchoData.from_file(converted_raw_path=converted_zarr)
-        ed['Sonar/Beam_group1'] = ed['Sonar/Beam_group1'].rename({'frequency': 'channel'})
+        ed['Sonar/Beam_group1'] = ed['Sonar/Beam_group1'].rename({'beam': 'beam_newname'})
 
-        assert sorted(ed['Sonar/Beam_group1'].dims.keys()) == ['channel', 'ping_time', 'range_bin']
+        assert sorted(ed['Sonar/Beam_group1'].dims.keys()) == ['beam_newname', 'channel', 'ping_time', 'range_sample']
```

Comment:
@emiliom did you add/modify these lines? I don't remember this being the current state of these tests. @lsetiawan's PR #665 that I approved is different here. Even stranger, somehow `range_bin` showed up here...

Reply:
This test failed, so I traced its origin. @lsetiawan added it on May 6. Line 328 was originally (in Don's PR) as:
```python
ed['Sonar/Beam_group1'] = ed['Sonar/Beam_group1'].rename({'frequency': 'channel'})
```
It turns out that the test was using one of the old (0.5.6), pre-converted test datasets (`*__NEW.zarr`) I mentioned above. That's why it referred to the `frequency` and `range_bin` dimension names that no longer exist. These files need to be updated to the current state of 0.6.0dev, so when I updated them, the test failed. Hence my modifications to the test.

Reply:
I see, ugh this is messy. I suppose this will be the case until we arrive at a final form for the structure.